src/main/java/org/openrewrite/java/migrate/apache/commons/io/ApacheCommonsFileUtils.java

```diff
@@ -47,5 +48,30 @@ File after(String name) {
 //        }
 //    }
 
+    private static class CopyDirectory {
+        @BeforeTemplate
+        void before(File a, File b) throws Exception {
+            FileUtils.copyDirectory(a, b);
+        }
+
+        @AfterTemplate
+        void after(File a, File b) throws Exception {
+            Files.copy(a.toPath(), b.toPath());
```

Comment:
I think the copy options `REPALCE_EXISTING` and probably also `COPY_ATTRIBUTES` should be specified to maintain a behavior as close as possible to that of `FileUtils`. However, in many cases users may not actually be really dependent on `COPY_ATTRIBUTES`, so I am unsure what to do about that one. Also, this is still just an approximation of what `FileUtils` does.

Reply:
If this is not a direct replacement, maybe best to leave it out for now @AlekSimpson